src/pages/DetailPage/DetailInfo.js

```diff
@@ -0,0 +1,79 @@
+import styled from 'styled-components';
+import Detailoption from './Detailoption';
+
+const DetailInfo = () => {
+  return (
+    <Wrapdiv>
+      <section>
+        <Flexdiv>
+          <Namespan>침대1</Namespan>
+          <div>
+            <img src="../images/ribbon.png" alt="ribbon" />
+            <img src="../images/share.png" alt="share" />
+          </div>
+        </Flexdiv>
+        <Pricediv>
+          <span>1,000</span>
+          <span>원</span>
+          <div>특가</div>
+        </Pricediv>
+        <Infodiv>
+          <Subspan>혜택</Subspan>
+          <span>78p 적립(WELCOME 0.3%적립)</span>
+        </Infodiv>
+        <Infodiv>
+          <Subspan>배송</Subspan>
+          <span>3,000원 (50,000원 이상 구매시 무료배송)</span>
+        </Infodiv>
+      </section>
+      <Detailoption />
+    </Wrapdiv>
+  );
+};
+
+const Flexdiv = styled.div`
+  display: flex;
+  div {
+    margin-left: 350px;
+  }
```

Comment:
div 태그 처럼 빈번히 사용되는 태그로 스타일을 주는 건 depth에 무관하게 적용되어 의도치 않은 스타일링이 될 수 있습니다.
```suggestion
  margin-left: 350px;
```
그냥 이렇게 줘도 같은 의미가 되지 않나요?

Reply:
수정했습니다!